.eslintrc.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   env: {
     node: true,
   },
+  // plugins: ['@renovate'],
```

Comment:
can we references this via giturl as dependency until we release it to npm / github pacakges?

Reply:
I'm kind of learning towards leaving it as git-only, because we will be the only user

Reply:
So we should add a `v0.0.1` tag to `renovatebot/eslint-plugin` and reference it in `package.json` as 

```json
 "@renovate/eslint-plugin": "renovatebot/eslint-plugin#v0.0.1"
```

We can also just use the commit hash for now.


```json
 "@renovate/eslint-plugin": "renovatebot/eslint-plugin#1ebc0b1"
```